test/e2e/l7networkpolicy_test.go

```diff
@@ -123,6 +128,17 @@ func createL7NetworkPolicy(t *testing.T,
 	assert.NoError(t, err)
 }
 
+func getPodIPs(podIPs *PodIPs) []*net.IP {
+	var ips []*net.IP
+	if podIPs.IPv4 != nil {
+		ips = append(ips, podIPs.IPv4)
+	}
+	if podIPs.IPv6 != nil {
+		ips = append(ips, podIPs.IPv6)
+	}
+	return ips
+}
```

Comment:
nit: maybe we could add a method for the `PodIPs` struct?
```go
func (ips *PodIPs) AsSlice() []*net.IP {
	var ips []*net.IP
	if podIPs.IPv4 != nil {
		ips = append(ips, podIPs.IPv4)
	}
	if podIPs.IPv6 != nil {
		ips = append(ips, podIPs.IPv6)
	}
	return ips
}
```

Reply:
Done

Reply:
Note sure this was addressed, did I miss something?

Reply:
oops I added the method with a different name, updated.